### PR TITLE
update v0.2.1 Readme

### PR DESCRIPTION
**Pull request by yume190**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -13,20 +13,22 @@ development inside a Bazel workspace.
...
-  * [1\. Configure your workspace to use <a href="https://github\.com/cgrindel/swift\_bazel">swift\_bazel</a>\.](#1-configure-your-workspace-to-use-swift_bazel)
-  * [2\. Create a minimal Package\.swift file\.](#2-create-a-minimal-packageswift-file)
-  * [3\. Add Gazelle targets to BUILD\.bazel at the root of your workspace\.](#3-add-gazelle-targets-to-buildbazel-at-the-root-of-your-workspace)
-  * [4\. Resolve the external dependencies for your project\.](#4-resolve-the-external-dependencies-for-your-project)
-  * [5\. Create or update Bazel build files for your project\.](#5-create-or-update-bazel-build-files-for-your-project)
-  * [6\. Build and test your project\.](#6-build-and-test-your-project)
-  * [7\. Check in some generated files\.](#7-check-in-some-generated-files)
-* [Tips and Tricks](#tips-and-tricks)
-* [Future Work](#future-work)
+- [Gazelle Plugin for Swift and Swit Package Rules for Bazel](#gazelle-plugin-for-swift-and-swit-package-rules-for-bazel)
```
Comment (cgrindel) [inline README.md:16]: We have [a utility and test](https://github.com/cgrindel/swift_bazel/blob/main/BUILD.bazel#L135) that generates the TOC for our markdown files. Did you see something that was not correct in the generated TOC?

File: README.md [hunk 2]
```diff
@@ -78,10 +80,9 @@ load("@bazel_tools//tools/build_defs/repo:http.bzl", "http_archive")
 
 http_archive(
     name = "cgrindel_swift_bazel",
-    sha256 = "fd77181e45fbb9ab6ddedf59f3f2d4cf0c173919a6de8d4a398d99fd965d5ce5",
-    strip_prefix = "swift_bazel-0.2.0",
+    sha256 = "2bcbe2947649f6433bf97258401c387eb41153c8adc378f84295628d879092d2",
     urls = [
-        "http://github.com/cgrindel/swift_bazel/archive/v0.2.0.tar.gz",
+        "https://github.com/cgrindel/swift_bazel/releases/download/v0.2.1/swift_bazel.v0.2.1.tar.gz",
```
Comment (cgrindel) [inline README.md:85]: The PR that automatically updates this section to the latest release was hung up by a GH actions issue. It is rolling through now. So, you will need to rebase in a few minutes.

**Review by cgrindel — COMMENTED** [review]
Thank you for the PR. Just a question about the TOC and you will need to rebase once #221 merges.

File: README.md (hunk 1 above)
Comment (yume190) [reply README.md]: sorry, my VSCode plugin change TOC.

**Review by cgrindel — APPROVED** [review]

**Pull request merged**